querydsl-core/src/main/java/com/querydsl/core/types/ConstantImpl.java

```diff
@@ -28,50 +28,52 @@
 
     private static final int CACHE_SIZE = 256;
 
-    @SuppressWarnings({"rawtypes", "unchecked"}) //generic array creation not possible
-    private static final Constant<Character>[] CHARACTERS = new Constant[CACHE_SIZE];
```

Comment:
Is it also possible to reorder these to the original ordering?
Or do you want it in this order?